solr/solrj/src/java/org/apache/solr/common/cloud/DocCollection.java

```diff
@@ -393,6 +411,8 @@ public String toString() {
         + getZNode()
         + "/"
         + znodeVersion
+        + "//"
```

Comment:
Do you think this stat is important enough to put itself in the toString?  I lean to "no".